src/style.css

```diff
@@ -1 +1,8 @@
-/* styles go here */
+.form-field {
+  margin-bottom: 10px;
+}
+
+fieldset {
```

Comment:
use classname instead of tag selector

Reply:
Not fixed 
<img width="541" alt="image" src="https://user-images.githubusercontent.com/99403198/216770814-f0ca4f38-cee6-4a62-9a8c-24637bf156f7.png">